ci/tasks/open-charts-pr.sh

```diff
@@ -30,3 +30,4 @@ gh pr create \
   --base ${BRANCH} \
   --head ${BOT_BRANCH} \
   --label galoybot
+  --label galoy
```

Comment:
You are missing a `\` at the end of the line above. In bash `\` means that the command continues on the next line.

Reply:
OMG yes, thanks for noticing this!